cypress/cypress.dev.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
we hve two `cypress.dev.json`, is this correct?

Reply:
Oops. Let me fix that. I had it in the wrong folder initially. Good catch!

Reply:
Removed it.